test/definition/ApiDefinitionFactorySpec.scala

```diff
@@ -37,8 +36,10 @@ class ApiDefinitionFactorySpec extends UnitSpec {
     "called" should {
       "return a valid Definition case class" in new Test {
         MockedAppConfig.featureSwitch returns None
-        MockedAppConfig.apiStatus returns "1.0"
-        MockedAppConfig.endpointsEnabled returns true
+        MockedAppConfig.apiStatus("1.0") returns "STABLE"
```

Comment:
does this not also need changing in application.conf? it's still alpha there, but I could be wrong here.

Reply:
These are just mocked values to check that the factory picks up them from config. I will update the test to make that clearer.